content/dev_stories/playground_to_production_fruit_falling.md

```diff
@@ -0,0 +1,677 @@
+    to fix it anyway.
+
+    ![Raw sprite location](/img/devVignettes/fruitFalling/10_raw_sprite_location.jpg!700)
+
+1.  After saving that change, all I had to do was save the file
+    and wait for the test app in my browser to auto-refresh.
+
+    ![It's working!](/img/devVignettes/fruitFalling/11_playground_test_app.jpg!500)
+
+And, just like that, I'd moved my prototype code from the Playground to 
+an independent repository in which to continue development. However, I 
+was still pulling assets from the Internet, which would be difficult to
+control and update as I continued developing my app. Time to create a 
+more development-friendly environment for assets too!
+
+### Creating an Asset Host Repo
+
+The `Sprite` I used while learning how to use sprites is the one provided
+as part of Babylon's default assets. While I knew all along I wanted to
+replace that, I first wanted to use it to bootstrap and test my approach
+to asset hosting for development.
+
+1.  Repeating essentially the steps from the 
+    [prior section](#creating-an-asset-host-repo) 
+    for creating a new repository from a template, I made a new asset host 
+    from the 
+    [Asset Host Template](https://github.com/BabylonJS/npm-package-template),
+    cloned it down, and installed the NPM dependencies.
+1.  As before, I ran
+    
+    ```
+    npm run dev
+    ```
+
+    in the root directory of my new asset host repo. This started up a 
+    server to serve assets in my repo's `docs` folder at 
+    http://127.0.0.1:8181 by default.
+
+    ![Asset server](/img/devVignettes/fruitFalling/12_asset_host_terminal.jpg!500)
+1.  To double-check that my server was working, I opened 
+    http://127.0.0.1:8181/example.txt and saw the text
+
+    ![Browser screenshot](/img/devVignettes/fruitFalling/13_example_txt.jpg!500)
+1.  Next, I downloaded a copy of 
+    [player.jpg](https://github.com/BabylonJS/Babylon.js/blob/master/Playground/textures/player.jpg)
+    and placed it in the `docs` folder. Again, I used the browser to test
+    that this worked by checking that I could see the image at
+    http://127.0.0.1:8181/player.jpg. The dev server that comes with the 
+    Asset Host Template automatically serves new files dropped in the 
+    `docs` folder, so there was no need to restart the server.
+
+    ![Player.jpg in the docs folder](/img/devVignettes/fruitFalling/14_assets_docs_folder.jpg!350)
+1.  Going back over to Playground.ts in my source repository, all that
+    remained to consume the sprite from my own asset host was to change
+    the the `SPRITE_URL` assignment line to
+
+    ```ts
+    const SPRITE_URL: string = "http://127.0.0.1:8181/player.jpg";
+    ```
+
+    save the file, then wait for the test app to reload.
+1.  Of course, once that reload happened, I realized I wouldn't be able
+    to see the results because the sprite from my asset host looked 
+    identical to the one pulled from the Internet. To make extra sure, 

```

Comment:
I think here, to be a little more direct, it would be better to recommend users to download a different asset from the start?

Reply:
I see what you mean, but I'm really trying to separate concerns between the major steps, so I don't want to talk about changing the app's behavior/visage while bringing it into a dev repo. The thing this is trying to emphasize is that you
can get your Playground working as-is in the dev repo, after which you can iterate on it. Suggesting using a different
asset probably wouldn't be _too_ big a deviation from that, but it would also remove the opportunity to bring up 
these common asset-serving pitfalls (wrong names, browser caching, etc.), which is what this step really exists to mention. I've confused myself quite a few times with names and browser-cached assets, so I specifically wanted to show that it's worth being mindful of things like that and knowing how to overcome them.

Reply:
That makes sense!